charts/e2e-resources/Chart.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v2
+version: 1.2.0
+appVersion: "1.0.0"
```

In case we release it by mistake. `appVersion` is not mandatory.
```suggestion
version: 1.2.0-devel
```